plugin.cmake

```diff
@@ -68,6 +69,15 @@ set_property(TEST server_large_image.cached_tiles.PythonCache APPEND PROPERTY
   "LARGE_IMAGE_CACHE_BACKEND=python"
   "LARGE_IMAGE_DATA=${PROJECT_BINARY_DIR}/data/plugins/large_image")
 
+add_python_test(sources PLUGIN large_image BIND_SERVER EXTERNAL_DATA
+  # I don't know why adding this data file thows a cmake error.
+  # "plugins/large_image/sample_image.ptif"
```

Comment:
Hm, I'm not super familiar with the internals of ExternalData. Brad King is the expert on that subject.